docs/cortex/installation-and-configuration/run-cortex-with-docker.md

```diff
@@ -1,72 +1,112 @@
 
 !!! Example ""
     ```
     docker run --volume /var/run/docker.sock:/var/run/docker.sock --volume /var/run/cortex/jobs:/tmp/cortex-jobs thehiveproject/cortex:latest --job-directory /tmp/cortex-jobs --docker-job-directory /var/run/cortex/jobs
     ```
 
-Cortex can instantiate docker container by using the docker socket `/var/run/docker.sock`. The folder `/var/run/cortex/jobs` is used to store temporary file of jobs. The folder `/tmp/cortex-jobs` is job folder inside the docker. In order to make job file visible to analyzer docker, Cortex needs to know both folders (parameters `--job-directory` and `-docker-job-directory`). On most cases, job directories are the same and `--docker-job-directory` can be omitted.
+Cortex creates Docker containers via the Docker socket `/var/run/docker.sock`. The directory `/var/run/cortex/jobs` stores temporary job files on the host, and `/tmp/cortex-jobs` is the corresponding path inside the container. Cortex requires both paths via `--job-directory` (inside container) and `--docker-job-directory` (host path) so job files are available to analyzers. Usually, when host and container paths are the same, `--docker-job-directory` can be omitted.
 
-If you run Cortex in Windows, the docker service is accessible through the named pipe `\\.\pipe\docker_engine`. The command becomes
+On Windows, Docker service is accessible through the named pipe `\\.\pipe\docker_engine`. The command changes accordingly:
 
 !!! Example ""
     ```
     docker run --volume //./pipe/docker_engine://./pipe/docker_engine --volume C:\\CORTEX\\JOBS:/tmp/cortex-jobs thehiveproject/cortex:latest --job-directory /tmp/cortex-jobs --docker-job-directory C:\\CORTEX\\JOBS
     ```
 
-#### Docker in docker (docker-ception)
-You can also run docker service inside Cortex container, a docker in a docker with `--start-docker` parameter. The container must be run in privileged mode.
+### Running Docker inside Cortex container (Docker-in-Docker)
+
+You can also run a Docker service inside the Cortex container itself—essentially Docker within Docker—by using the `--start-docker` parameter. Note that you must start the container in privileged mode.
 
 !!! Example ""
     ```
     docker run --privileged thehiveproject/cortex:latest --start-docker
     ```
-In this case you don't need to bind job directory.
 
-#### Use Docker-compose
-Cortex requires Elasticsearch to run. You can use `docker-compose` to start them together in Docker or install and configure Elasticsearch manually.
-[Docker-compose](https://docs.docker.com/compose/install/) can start multiple dockers and link them together.
+In this mode, you don’t need to bind any job directories since the Docker daemon runs inside the container.
+
+### Using Docker Compose to start Cortex and Elasticsearch
```

Comment:
Shouldn't we reference the [official docker compose repository](https://github.com/StrangeBeeCorp/docker)?